.eslintrc.js

```diff
@@ -32,6 +43,12 @@ module.exports = {
         document: true
       }
     },
+    {
+      files: ['src/**/*.tsx'],
+      rules: {
+        'react/prop-types': 'off'
```

Comment:
We're going to be removing react prop-types from TS components, so turn this rule off for `.tsx` files.